lib/src/models/definitions.ts

```diff
@@ -19,6 +20,11 @@ export interface ApiDefinition {
   securityHeader?: SecurityHeaderDefinition;
 }
 
+export const defaultConfigDefinition = { paramSerializationStrategy: {} };
```

Comment:
I added this constant here instead of using `{ paramSerializationStrategy: {} }` everywhere.

Let me know if there is an issue with this or if you have a suggestion.